resources/dicts/events/nutrition/leader.json

```diff
@@ -8,7 +8,7 @@
             "Leaf-bare",
             "death"
         ],
-        "death_text": "m_c feels that {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} have enough energy to stand up and remain here.",
+        "death_text": "m_c feels that {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} have enough energy to stand up and remain here as starvation claws into {PRONOUN/m_c/poss} belly and sends {PRONOUN/m_c/object} to {PRONOUN/m_c/poss} ancestral hunting grounds.",
```

Comment:
Be careful here, you add this line to the leader, but the leader might still have multiple life's left. @Tiritirimatangi you have the lead in the writing part.

Reply:
I think we should delete the "and sends {PRONOUN/m_c/object} to {PRONOUN/m_c/poss} ancestral hunting grounds". Hopefully soon an event rewrite will allow us to separate regular deaths and leader deaths like the patrols can do!

Reply:
@yippeecore